react/src/routes/home.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { TabNavigation } from '../components/navigation.js';
+import { FalconApiContext } from '../contexts/falcon-api-context.js';
+import { SlButton } from '@shoelace-style/shoelace/dist/react';
+function Home() {
+  const { isInitialized } = useContext(FalconApiContext);
+
+
+  if (!isInitialized) {
+    return 'Loading...';
+  }
+
+  return (
+    <TabNavigation>
+      <div className={`mt-4 space-y-8`} style={{ maxWidth: '800px' }}>
```

Comment:
React noob question: Is that odd `className` thing even required for static markup?